examples/scripts/iframe/index.mustache

```diff
@@ -8,27 +8,38 @@
     <script src="../../build/playcanvas-extras.js"></script>
     <script src="../../build/playcanvas-observer.js"></script>
     <script src="../../build/wasm-loader.js"></script>
+    <link rel="stylesheet" href="/styles.css">
     <style>
         body {
             margin: 0;
             overflow-y: hidden;
         }
-        #app-canvas {
+        #application-canvas {
             width: 100%;
             height: 100%;
         }
     </style>
 </head>
 
 <body>
-    <canvas id='app-canvas'></canvas>
+    <div id="app">
+        <div id="appInner">
+            <canvas id='application-canvas'></canvas>
+        </div>
+    </div>
     <script>
         {{{ exampleConstValues }}}.forEach(function(s) {
             window[s.k] = s.v;
         });
 
         function loadExample(exampleFunction, loadFunction, controlsFunction) {
-            var data = new observer.Observer({});
+            var data;
```

Comment:
Lots of vars in this file. Is that because of IE11? If so, you have arrow functions too....plus a const.

Reply:
Yep these files should support IE11. I've made the changes necessary so i'll give this a test.